Makefile

```diff
@@ -256,9 +256,10 @@ swagger-gen:
 swagger-docs: ## preview the API documentation
 	@echo "API docs preview will be running at http://localhost:$(SWAGGER_DOCS_PORT)"
 	@docker run --rm -v $(PWD)/api/swagger.yaml:/usr/share/nginx/html/swagger.yaml \
+        -e 'SPEC_URL=swagger.yaml' \
 		-e 'REDOC_OPTIONS=hide-hostname="true" lazy-rendering' \
 		-p $(SWAGGER_DOCS_PORT):80 \
-		bfirsh/redoc:1.14.0
+		redocly/redoc
```

Comment:
do they have version tags? might be good to pin to the same version as we'll be using in our docs (see my other comment)